src/project/ProjectNameEditable.tsx

```diff
@@ -61,6 +62,12 @@ const ProjectNameEditable = () => {
       >
         {project.name}
       </Text>
+      <ZoomControls
+        display={["none", "none", "none", "flex"]}
+        zIndex="1"
+        right={10}
+        position="absolute"
```

Comment:
Can we replace this with a more regular layout (e.g. a HBox with space-between).